src/form.js

```diff
@@ -983,6 +983,21 @@ define([
             var _this = this;
             this._logicManager.forEachBrokenReference(updateReferences);
         },
+        isReferencedByOtherMugs: function (mug, except) {
```

Comment:
After submitting that, I think maybe I didn't understand your suggestion. Help me understand if I missed your point.